pkg/grpc_testing/stub_server.go

```diff
@@ -93,20 +95,23 @@ func (ss *StubServer) Addr() string {
 
 type dummyStubServer struct {
 	testpb.UnimplementedTestServiceServer
-	body []byte
+	body    []byte
+	counter uint64
 }
 
-func (d dummyStubServer) UnaryCall(context.Context, *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+func (d *dummyStubServer) UnaryCall(context.Context, *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	newCount := atomic.AddUint64(&d.counter, 1)
+
 	return &testpb.SimpleResponse{
 		Payload: &testpb.Payload{
 			Type: testpb.PayloadType_COMPRESSABLE,
-			Body: d.body,
+			Body: []byte(strconv.FormatUint(newCount, 10)),
```

Comment:
It seems that the field `dummyStubServer.body` isn't used at all? If yes, then I suggest to remove it.

Reply:
Good catch, updated.  It was a little while back I worked on this but I think I was nervous about changing as I wasn't sure if there was going to be other reuse that depended on it.  After looking closer it seems ok to remove.